modules/apps/object/object-web/src/main/java/com/liferay/object/web/internal/object/entries/frontend/data/set/view/table/ObjectEntriesTableFDSView.java

```diff
@@ -124,8 +130,70 @@ else if (Objects.equals(objectField.getDBType(), "Date")) {
 		return fdsTableSchemaBuilder.build();
 	}
 
+	private String _getFieldName(ObjectField objectField) {
+		if (objectField.getListTypeDefinitionId() > 0) {
+			return objectField.getName() + ".name";
+		}
+		else if (Objects.equals(
+					objectField.getRelationshipType(), "oneToMany")) {
+
+			ObjectRelationship objectRelationship =
+				_objectRelationshipLocalService.
+					fetchObjectRelationshipByObjectFieldId2(
+						objectField.getObjectFieldId());
+
+			ObjectDefinition objectDefinition =
+				_objectDefinitionLocalService.fetchObjectDefinition(
+					objectRelationship.getObjectDefinitionId1());
+
+			ObjectField titleObjectField =
+				_objectFieldLocalService.fetchObjectField(
+					objectDefinition.getTitleObjectFieldId());
+
+			String relationshipName =
+				ObjectRelationshipNameUtil.getRelationshipName(
+					objectField.getName());
+
+			return relationshipName + StringPool.PERIOD +
+				titleObjectField.getName();
+		}
+
+		return objectField.getName();
+	}
+
+	private List<ObjectField> _getObjectFields() {
+		ObjectView defaultObjectView =
+			_objectViewLocalService.getFirstDefaultObjectView(
+				_objectDefinition.getObjectDefinitionId());
+
+		if (defaultObjectView == null) {
+			return _objectFieldLocalService.getObjectFields(
+				_objectDefinition.getObjectDefinitionId());
+		}
+
+		List<ObjectViewColumn> objectViewColumns =
+			defaultObjectView.getObjectViewColumns();
+
+		Stream<ObjectViewColumn> objectViewColumnsStream =
+			objectViewColumns.stream();
+
```

Comment:
@rodrigopaulino it could be just `stream` here, like you did in `ViewObjectEntriesDisplayContext` 

```
Stream<ObjectField> stream = objectFields.stream();
```